release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -863,8 +870,18 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 ==== {op-system-first}
 
 [discrete]
-[id="ocp-4-12-performance-addon-operator-bug-fixes"]
-==== Performance Addon Operator
+[id="ocp-4-12-low-latency-tuning-bug-fixes"]
+==== Low latency tuning
+
+* The Tuned profile has been modified to assign the same priority as `ksoftirqd` and `rcuc` to the newly introduced per-CPU kthreads (`ktimers`) added in a recent Red Hat Enterprise Linux (RHEL) kernel patch. For more information, see link:https://issues.redhat.com/browse/OCPBUGS-3475[*OCPBUGS-3475*], link:https://bugzilla.redhat.com/show_bug.cgi?id=2117780[*BZ#2117780*] and link:https://bugzilla.redhat.com/show_bug.cgi?id=2122220[*BZ#2122220*].
+
+* Previously, restarts of the `tuned` service caused improper reset of the `irqbalance` configuration, leading to IRQ operation being served again on the isolated CPUs, therefore violating the isolation guarantees. With this fix, the `irqbalance` service configuration is properly preserved across `tuned` service restarts (explicit or caused by bugs), therefore preserving the CPU isolation guarantees with respect to IRQ serving. For more information, see link:https://issues.redhat.com/browse/OCPBUGS-585[*OCPBUGS-585*].
+
+* Previously, when the Tuned daemon was restarted out of order as part of the cluster Node Tuning Operator, the CPU affinity of interrupt handlers was reset and the tuning was compromised. With this fix, the `irqbalance` plug-in in Tuned is disabled, and {product-title} now relies on the logic and interaction between `CRI-O` and `irqbalance`. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2105123[*BZ#2105123*].
+
+* Previously, the rollout times for `kube-apiserver` on loaded clusters were slow and sometimes exceeded the 5-minute rollout timeout. With this fix, the rollout times should be shorter and within the 5-minute threshold. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2109965[*BZ#2109965*].
```

Comment:
This bug is a symtom only of a bigger issue that we fixed. I think we should focus more on the underlying issue in the RN. Please check out the PR description of https://github.com/openshift/cluster-node-tuning-operator/pull/479 - simply said, we were executing a hook script for every new veth device created during a pod startup. And the internal logic took too long when the node was under load. These delays accumulated during pod start events which lead to the mentioned bug (timeout during rollout).

@Tal-or Could you please work with Kevin on the https://bugzilla.redhat.com/show_bug.cgi?id=2109965 RN and describe the cause, effect, fix, outcome as the initial template suggested please?

Reply:
@Tal-or Can you review the updated description please.